zk-sdk/src/sigma_proofs/percentage_with_cap.rs

```diff
@@ -10,12 +10,13 @@
 //! max cap value.
 //!
 //! A more detailed description of the context and how the proof is computed is provided in the
-//! [`ZK Token proof program`] documentation.
+//! [`ZK Token proof program`] documentation, specifically the percentage-with-cap [`specification`].
 //!
 //! The protocol guarantees computational soundness (by the hardness of discrete log) and perfect
 //! zero-knowledge in the random oracle model.
 //!
 //! [`ZK Token proof program`]: https://docs.solanalabs.com/runtime/zk-token-proof
```

Comment:
This can be done in future work, but I just noticed the `ZK Token Proof program` references :sweat_smile:

Reply:
Oh oops! I'll fix this in the next follow up!